### PR TITLE
Update README.md

### LINKED ISSUE #6795: Update REadme .MD
## Problem Definition

removed Grammatical errors and errors
### System Description

Operating system: 
Rotki version:

### PR DESCRIPTION
**Pull request by Shivam250702**
Removed Grammatical errors and punctuation marks errors

Closes 
#6795 
## Checklist

- [x] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,13 +69,13 @@ The [Pro Version](https://rotki.com/checkout/plan) of Rotki also supports additi
...
 
-The installation for rotki can be simply done via two ways:-
+The installation of rotki can be simply done in two ways:-
 - [Packaged Binaries](https://rotki.readthedocs.io/en/latest/installation_guide.html#packaged-binaries) (Recommended)
 - [Building from Source](https://rotki.readthedocs.io/en/latest/installation_guide.html#build-from-source)
 
 ## Usage
 
-Read the [documentation](https://rotki.readthedocs.io/en/latest/) for a complete guide, frequently-asked questions, API reference, and more.
+Read the [documentation](https://rotki.readthedocs.io/en/latest/) for a complete guide, frequently-asked questions, API references, and more.
```
Comment (LefterisJP) [inline README.md:78]: This is correct. Please do not change

File: README.md [hunk 2]
```diff
@@ -69,13 +69,13 @@ The [Pro Version](https://rotki.com/checkout/plan) of Rotki also supports additi
 ## Installation
 Rotki currently supports the following operating systems:- Windows, MacOs, Linux.
 
-The installation for rotki can be simply done via two ways:-
+The installation of rotki can be simply done in two ways:-
```
Comment (LefterisJP) [inline README.md:72]: The via is fine. Can also be in. The for/of is a correct change.

File: README.md [hunk 3]
```diff
@@ -40,13 +40,13 @@
 ---
 
 ## Overview
-[Rotki](https://rotki.com/) is an open source portfolio tracking, analytics, accounting, and tax reporting tool that protects your privacy.  The mission of rotki is to bring transparency into the crypto and financial sectors through the use of open source. Most importantly, unlike virtually every other competing service which consists of closed source SaaS onto which you are forced to hand over all your financial data, with rotki your data is stored encrypted locally in your computer. It enables you to take ownership of your financial data!
+[Rotki](https://rotki.com/) is an open source portfolio tracking, analytics, accounting, and tax reporting tool that protects your privacy.  The mission of rotki is to bring transparency into the crypto and financial sectors through the use of open source. Most importantly, unlike virtually every other competing service, which consists of closed source SaaS onto which you are forced to hand over all your financial data, with rotki your data is stored encrypted locally on your computer. It enables you to take ownership of your financial data!
```
Comment (LefterisJP) [inline README.md:43]: The `,` change is not needed

**Comment by Shivam250702** [comment]
Ok sir ,I will create a new PR with the underlying changes

**Comment by LefterisJP** [comment]
> Ok sir ,I will create a new PR with the underlying changes

No no no please just force push into this PR. You should not be opening a new PR to make changes into an existing PR. This is spamming our repository and increasing issues/PR numbers.

**Pull request closed without merging**